commons/admin-console/ssh-keys.md

```diff
@@ -40,7 +40,7 @@ You may already have an SSH key and so do not need to generate a new one. To che
 
 1. Wether you use Mac or Linux, open your Terminal application.
 2. Run `cd ~/.ssh/` in your Terminal.
-3. If the folder exists, run `ls` and check if a pair of key exists : *id_rsa* and *id_rsa.pub*.
+3. If the folder exists, run `ls` and check if a pair of key exists : *id_ed25519* and *id_ed25519.pub*.
```

Comment:
I think it would still be better to add a note here saying that you can also use a >X bits RSA key if you already have one even though it's *deprecated*. (same thing for the next patch chunk obviously)

Reply:
Added a reference to RSA just below!